cogs/fun.py

```diff
@@ -108,9 +115,22 @@ async def on_message(self, message):
 
         # Store msg if msg was sent in DM and turn on "confirm mode"
         if not message.guild:
-            self.confession_is_confirm_e = True
-            self.confession_msg = message
-            await message.channel.send("React to this message to send it to the confession channel")
+            pattern = self.regex_patterns.get(str(message.channel.id))
+
+            # Check if one or more emojis are in the msg content
+            emoji_in_msg = bool(re.search(pattern, message.content))
+
+            emoji_in_msg += sum(char in UNICODE_EMOJI["en"]
```

Comment:
If you set `emoji_in_msg` as bool, don't change his type afterwards, use proper bool operators.
```suggestion
            emoji_in_msg |= any(char in UNICODE_EMOJI["en"]
```